docs/content/en/contribute/software/dev-environ/_index.md

```diff
@@ -1,20 +1,79 @@
 ---
-title: Set up the development environment
-description: How to set up an environment to develop and test Keptn software
+title: Software development environment
+description: How to set up and use the development environment to develop and test Keptn software
 weight: 30
 ---
 
+This page gives instructions and hints for setting up a development environment
+and then develop, test, and deploy your software changes in that environment.
+This material was presented at the
+11 September 2023 New Contributors meeting.
+You can view the video
+[here](https://www.youtube.com/watch?v=UcmULstMYXQ).
+
+To prepare to contribute to the Keptn project, we recommend that you:
+
+* Study the [Keptn documentation](https://lifecycle.keptn.sh/docs/)
+  to understand what Keptn does and how it works.
+* Familiarize yourself with the
+  [lifecycle-toolkit](https://github.com/keptn/lifecycle-toolkit)
+  repository, which is the primary repository for
+  Keptn software and documentation.
+  In particular, study the sections for the four main Keptn components:
+  
+  * [lifecycle-operator](https://github.com/keptn/lifecycle-toolkit/tree/main/lifecycle-operator)
+  * [metrics-operator](https://github.com/keptn/lifecycle-toolkit/tree/main/metrics-operator)
+  * [scheduler](https://github.com/keptn/lifecycle-toolkit/tree/main/scheduler)
+  * [klt-cert-manager](https://github.com/keptn/lifecycle-toolkit/tree/main/klt-cert-manager)
+
+  Each of these is described in the
+  [Architecture](../../../docs/architecture/)
+  section of the documentation
+  and most include a *README* file with more information.
+* Study the material in
+  [Technologies and concepts you should know](../../general/technologies).
+* Create an account for yourself on
+  [GitHub](https://github.com)
+  if you do not already have an account.
+* Set up a fork of the [lifecycle-toolkit](https://github.com/keptn/lifecycle-toolkit) repository to use with your development.
+
+## View repository
+
+When you view the
+[lifecycle-toolkit](https://github.com/keptn/lifecycle-toolkit)
+repository, you see that Keptn is composed of multiple components,
+each of which is discussed in the Architecture
+[Architecture](../../../docs/architecture/)
+documentation:
+
+* Two Kubernetes operators, `metrics-operator` and `lifecycle-operatory`
+* Keptn `scheduler`
+* Keptn `cert-manager`
```

Comment:
The `cert-manager` is also an operator, so we can mention it in the first bullet point, next to the other two operators

Reply:
Who knew?  It's fixed.